server/views/applications/add.njk

```diff
@@ -4,7 +4,7 @@
 
 {% from "govuk/components/details/macro.njk" import govukDetails %}
 
-  <a href="/manage-applications" class="govuk-back-link">Back to all applications</a>
+  {# <a href="/manage-applications" class="govuk-back-link">Back to all applications</a> #}
```

Comment:
Why are these links commented out?

Reply:
Because of the move to remove these back links to a shared component if I left them in the links would double up.
I left them uncommented rather than deleted as my story is update key rather than any other so I think its the responsibility of other related tickets to verify the back link functionality is still behaving as expected.